code/game/objects/effects/spiders.dm

```diff
@@ -31,7 +31,10 @@
 	take_damage(5, BURN, 0, 0)
 
 /obj/structure/spider/stickyweb
+	///Whether or not the web is from the genetics power
 	var/genetic = FALSE
+	///Whether or not the web is a sealed web
+	var/sealed = FALSE
 	icon_state = "stickyweb1"
```

Comment:
In the future please put new variables below existing ones (I know genetics is also an offender but double cringe)